bundles/AdminBundle/Controller/Admin/DataObject/DataObjectHelperController.php

```diff
@@ -1619,6 +1619,12 @@ public function batchAction(Request $request)
 
                                 /** @var DataObject\Classificationstore $classificationStoreData */
                                 $classificationStoreData = $object->$getter();
+                                if ($append) {
+                                    $oldValues = $classificationStoreData->getLocalizedKeyValue($groupId, $keyId);
+                                    if(is_array($oldValues)) {
+                                        $value = array_merge($oldValues, $value);
+                                    }
+                                }
```

Comment:
we should use `appendData` (to avoid duplicate values) and `removeData` (to remove specific value)
```suggestion
if ($append) {
                                    $oldValue = $classificationStoreData->getLocalizedKeyValue($groupId, $keyId);
                                    $value = $dataDefinition->appendData($oldValue, $value);
                                }
                                if ($remove) {
                                    $oldValue = $classificationStoreData->getLocalizedKeyValue($groupId, $keyId);
                                    $value = $dataDefinition->removeData($oldValue, $value);
                                }
```